sources/TileDB.CSharp/File.cs

```diff
@@ -1,172 +1,119 @@
 using System;
-using System.Text;
 using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using TileDB.Interop;
 
 namespace TileDB.CSharp
 {
-    public class File
+    public sealed unsafe class File
```

Comment:
Since the API is still early, suggest to consider harmonizing the name to `Filestore` for consistency with core?